<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>851</width>
      <height>140</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QFormLayout" name="LayoutInfinito">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Planetas:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>X:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="rX">
                   <property name="minimum">
                    <double>-1000000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>1000000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Y:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="rY">
                   <property name="minimum">
                    <double>-1000000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>1000000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Vx:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="rVx">
                   <property name="minimum">
                    <double>-1000000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>1000000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Vy:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="rVy">
                   <property name="minimum">
                    <double>-1000000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>1000000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Masa:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="rMasa">
                   <property name="maximum">
                    <double>1000000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Radio:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="rRadio">
                   <property name="decimals">
                    <number>2</number>
                   </property>
                   <property name="maximum">
                    <double>1000000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="ZoomPlus">
               <property name="text">
                <string>Zoom+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ZoomMinus">
               <property name="text">
                <string>Zoom-</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="Iniciar">
             <property name="text">
              <string>Iniciar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Detener">
             <property name="text">
              <string>Detener</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Ingresar">
             <property name="text">
              <string>Ingresar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="numPlanets">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Left">
         <property name="text">
          <string>Left</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="Up">
           <property name="text">
            <string>Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Down</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="Right">
         <property name="text">
          <string>Right</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>graphicsView</tabstop>
  <tabstop>ZoomPlus</tabstop>
  <tabstop>ZoomMinus</tabstop>
  <tabstop>rY</tabstop>
  <tabstop>rVx</tabstop>
  <tabstop>rVy</tabstop>
  <tabstop>rMasa</tabstop>
  <tabstop>rRadio</tabstop>
  <tabstop>rX</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
